notebooks/01_prepare_data/wikidata.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>in the other notebooks we have an initial cell where we put the `MOVIELENS_DATA_SIZE`, see here: <a href="https://github.com/microsoft/recommenders/blob/master/notebooks/00_quick_start/sar_movielens.ipynb" target="_blank">https://github.com/microsoft/recommenders/blob/master/notebooks/00_quick_start/sar_movielens.ipynb</a>. We use that cell to programatically inject variables through papermill  for the tests. See  <a href="https://github.com/microsoft/recommenders/tree/master/tests#how-to-create-tests-on-notebooks-with-papermill" target="_blank">https://github.com/microsoft/recommenders/tree/master/tests#how-to-create-tests-on-notebooks-with-papermill</a></p>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/778/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Implemented here: 9c03398